Makefile

```diff
@@ -3,7 +3,8 @@
 VERSION	:= $(shell if test -f manifest.wake; then sed -n "/publish releaseAs/ s/^[^']*'\([^']*\)'.*/\1/p" manifest.wake; else git describe --tags --dirty; fi)
 
 CC	:= cc -std=c11
-CXX	:= c++ -std=c++11
+CXX	:= c++
+CXX_VERSION := -std=c++14
 CFLAGS	:= -Wall -O2 -DVERSION=$(VERSION)
```

Comment:
nit: I think just adding the CXX_VERSION onto the end of CFLAGS would be better

Reply:
Happy to move it, but CORE_CFLAGS is listed after CFLAGS in the build rule so it would need to be pushed back as well. Is that ok?

Reply:
Also CFLAGS is used for the C build rules

Reply:
Ah I see. I guess a name like CXX_FLAGS would be better then in case we need more overrides in the future. All good as is.